.circleci/config.yml

```diff
@@ -4,169 +4,166 @@ jobs:
     # must be IROHA_HOME:
     working_directory: /opt/iroha
     docker:
-      - image: warchantua/iroha-dev
+      - image: lebdron/iroha-dev
```

Comment:
Maybe we can only uncomment this https://github.com/hyperledger/iroha/blob/develop/.circleci/config.yml#L93
and we can use `hyperledger/iroha-docker:develop`